src/main/java/com/mattermost/integration/figma/input/file/notification/FileCommentWebhookResponse.java

```diff
@@ -0,0 +1,44 @@
+package com.mattermost.integration.figma.input.file.notification;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.*;
+
+@Getter
+@Setter
```

Comment:
pls take a look on lombok "@ Data" annotation I believe it aggregates most of this annotations